src/vignette/api/legacy/routes.clj

```diff
@@ -49,6 +50,20 @@
                   :image-type #"images"
                   :original original-regex}))
 
+(def interactive-maps-route
+  (route-compile "/:wikia:path-prefix/:original"
+                 {:wikia interactive-maps-regex
+                  :path-prefix #"/\d/\d|"
```

Comment:
The first number currently can be [in range 0-8](https://github.com/Wikia/config/blob/dev/InteractiveMapsConfig.json#L144) but the second one might be higher. It depends on the image size for given zoom. For example /5/23/9.png is a valid path.